pkg/scm/scm.go

```diff
@@ -45,8 +45,7 @@ func DownloaderForSource(fs util.FileSystem, s string, forceCopy bool) (build.Do
 		return &file.File{FileSystem: fs}, s, nil
 	}
 
-	// If the source is valid  Git protocol (file://, ssh://, git://, git@, etc..) use Git
```

Comment:
> the old one at least brought to mind some of the potential options the code might be dealing with.

IMHO there are better places to tell what are valid clone specs:

- ValidCloneSpec doc comment
- Tests
- Git documentation

I left the comment as a marker of "here is where we decide we'll try a `git clone`", I'm not so concerned with enumerating, in this particular place, all the protocols we might recognize.

Further reasoning, if we add or remove support for a certain protocol, do you think this is an obvious place where we would update the comment, or it would easily drift?